python/variable.cpp

```diff
@@ -181,12 +180,6 @@ of variances.)");
       py::arg("x"), py::arg("dim") = py::none(),
       py::call_guard<py::gil_scoped_release>());
 
```

Comment:
is the variable level `rebin` with this overload still required? I notice that this was not removed alongside the python binding.

Reply:
Yes, it is used in the C++ implementation of `rebin` for data arrays.